giftcard/src/main/java/com/adyen/checkout/giftcard/internal/ui/GiftCardDelegate.kt

```diff
@@ -41,4 +41,6 @@ internal interface GiftCardDelegate :
     fun resolveBalanceResult(balanceResult: BalanceResult)
 
     fun resolveOrderResponse(orderResponse: OrderResponse)
+
+    fun isPinRequired(): Boolean
```

Comment:
Would this be better as part of the `GiftCardOutputData`?

Reply:
Hmm maybe, but then we would potentially do unnecessary initialisation when creating `GiftCardView`. I don't really have a strong preference, do you?

Reply:
Not really, up to you 👍

Reply:
Okay then I'll keep it this way